tensorflow/lite/micro/kernels/xtensa_hifi/xa_nnlib/algo/common/include/xa_nnlib_common_macros.h

```diff
@@ -0,0 +1,699 @@
+
+/******************************************************************************
+* Copyright (C) 2019 Cadence Design Systems, Inc.
```

Comment:
These copyrights will need to be converted to the standard "TensorFlow Authors" license used elsewhere.

Reply:
Hi Pete,

We like to host our xa_nnlib source code same as Nick has done for hifi_mini library and then at later point of time we like to host our Cadence github which will allow to download the xa_nnlib independently. 
https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/micro/kernels/xtensa-hifimini

Copyright, we can change to the standard "TensorFlow Authors" license

Thanks,
Niranjan

Reply:
We’ll have full NN-lib on “Cadence github”. Whereas, under TFLite Micro git-hub, we’ll have a copy of subset of NN-lib operators relevant to TF Lite Micro.